packages/labs/analyzer/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@lit-labs/analyzer",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "A static analyzer for Lit",
+  "license": "BSD-3-Clause",
+  "author": "Google LLC",
+  "homepage": "https://github.com/lit/lit/tree/main/packages/labs/analyzer",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/labs/analyzer"
+  },
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc --build",
+    "install-test-deps": "cd test-files/basic-elements && npm ci",
```

Comment:
it may be enough to express "before running tests, if any of the package.jsons in test-files have changed, then install test deps"

Reply:
Maybe a path `devDependency` dependency would work? Not sure if it matters that that could cause some dependencies to be installed/deduped into `analyzer/node_modules` instead of `basic-elements/node_modules`?

Reply:
that did work. It did also hoist `@lit/...` into the analyzer's node_modules, but I think that's ok.